server/views/pages/coordinator/staff-member-not-removed.html

```diff
@@ -32,7 +32,7 @@ <h2 class="govuk-heading-l">
           </div>
       </dl>
 
-      <p><a href={{data.returnToIncidentLink}}>Return to use of force incident</a></p>
+      <p><a href="/{{ data.reportId }}/view-statements">Return to use of force incident</a></p>
```

Comment:
not related to this PR, I had a thought we could make the links object as part of the `app.locals` (or `res.locals`) and then use it in templates too: `href="{{ links.viewStatement(data.reportId) }}` or the like